src/video_core/engines/shader_bytecode.h

```diff
@@ -981,6 +981,10 @@ union Instruction {
             }
             return false;
         }
+
+        bool IsComponentEnabled(std::size_t component) const {
+            return ((1ul << component) & component_mask) != 0;
```

Comment:
```suggestion
            return ((1ULL << component) & component_mask) != 0;
```

unsigned long is 32-bit on Windows. (It's also better to uppercase literals, since `L` can't be mistaken for a `1` depending on the font used.